src/ShallowWrapper.js

```diff
@@ -80,8 +80,16 @@ export default class ShallowWrapper {
           }
         });
       });
-      this.node = this.renderer.getRenderOutput();
-      this.nodes = [this.node];
+      Object.defineProperty(this, 'node', {
+        get() {
```

Comment:
The requirement to manually `update()` is intentional. I agree it's a UX improvement to do it automatically, but it's not a bug.